gatsby-config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     title: `Cloudflare Workers`,
     description: `Use Cloudflare’s APIs and edge network to build secure, ultra-fast applications.`,
     author: `@cloudflaredev`,
+    siteUrl: `https://developers.cloudflare.com/`,
```

Comment:
Do we want to provide a sitemap for `https://developers.cloudflare.com/`, or for `https://developers.cloudflare.com/workers/` ?